_pages/SDK/v7.1/022-basics/0225-basics-security.md

```diff
@@ -49,3 +50,23 @@ DataServiceClient.AuthenticationContainer = authContainer;
 {% endhighlight %}
 This tells the ServiceClient to use Windows Authentication (ActiveDirectory). From now on the client will use your Windows user account for authentication. You do not have to specify credentials as the client will check and fetch your information from the organization directory. <br>
 Remember that your authentication mode has to match the mode set in the server settings.
+
+{{ site.headers['example'] }} Authenticate using OpenID connect
+
+{% highlight csharp %}
+// use OAuthHelper to get the OIDC access token for the service
+var oAuthTokenCredential = OAuthHelper.GetAuthenticationInformationForDatabaseUrl(serverUri.AbsoluteUri, requestCallback: OAuthRequestCallback);
```

Comment:
Should we encourage the use of the async variant, so GetAuthenticationInformationForDatabaseUrlAsync instead?

Reply:
done